src/main/java/com/semmle/jira/addon/LgtmServlet.java

```diff
@@ -44,13 +44,30 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
     String[] pathParts = pathInfo.split("/");
     String configKey = pathParts[1];
 
+    Config config = Config.get(configKey);
+    List<Error> configErrors = config.validate();
+    for (Error error : configErrors) {
```

Comment:
This is a bit strange validate returns a list of errors, but we only tell the user about the first.